src/common/socket.c

```diff
@@ -238,7 +238,11 @@ fd_set readfds;
 
 // Maximum packet size in bytes, which the client is able to handle.
 // Larger packets cause a buffer overflow and stack corruption.
-static size_t socket_max_client_packet = 24576;
+#if PACKETVER >= 20131223
+static size_t socket_max_client_packet = 0x10000;
```

Comment:
When I looked in the client through ollydbg, I saw the hex value as 0x10000 and just added this as 65536. Older clients showed a value of 0x5000.